.agent-versions.json

```diff
@@ -1,9 +1,10 @@
 {
   "testVersions": [
+    "8.15.1-SNAPSHOT",
     "8.15.0",
-    "8.15.0-SNAPSHOT",
     "8.14.4-SNAPSHOT",
     "8.14.3",
-    "7.17.24-SNAPSHOT"
+    "7.17.24-SNAPSHOT",
+    "7.17.23"
```

Comment:
I don't think we want this line in here, given https://github.com/elastic/elastic-agent/pull/5270, right @michel-laterman @blakerouse?

Reply:
Correct, this line needs to be removed until 7.17.24 is released and it becomes:

```
7.17.25-SNAPSHOT
7.17.24
```